### PR TITLE
Hassbian docs

### PR DESCRIPTION
**Pull request by ludeeus**
**Description:**
Updated docs to better match the current version of the tool `hassbian-config`

### CODE REVIEW

File: source/_docs/installation/hassbian/common-tasks.markdown [hunk 1]
```diff
@@ -36,6 +36,10 @@ To get the current state of the `homeassistant.service` replace `stop` with `sta
 
 ### {% linkable_title Update Home Assistant %}
 
+<p class='note'>
+You can aslo so the atomated way with "hassbian-config" by running "sudo hassbian-config upgrade home-assistant"
```
Comment (fabaff) [inline source/_docs/installation/hassbian/common-tasks.markdown]: Fix the typos and please don't remove the formatting.

File: source/_docs/installation/hassbian/customization.markdown [hunk 2]
```diff
@@ -10,18 +10,27 @@ footer: true
...
- - Install Open Z-Wave. Installs Python Open Z-Wave and OZWCP from git.
+ - Install Libcec. Adds local [HDMI CEC support][cec]. *This scipt is currently brooken upstream since it currently doesn't build properly for Python >3.4*
  - Install Samba. Allows anyone on your network to edit your configuration from any computer. This share is unsecured and it's usage is not recommended if you share your network with others.
- - Install Tellstick. Installs the Tellstick package for controlling and using a connected Tellstick.
  - Install Tradfri. Installs dependencies for using IKEA Trådfri.
+ - Install Duck DNS auto renewal. This script adds an cron job to auto update you the WAN IP address for the defined domain. Before running this script you should already have an Duck DNS account, during the installation you will be asked to supply your domain name and the token for your account.
+ - Install an web terminal for easy access to ssh in an web browser. This script installs an web terminal called 'shellinabox' to you system that give you SHH access in you web browser.
```
Comment (fabaff) [inline source/_docs/installation/hassbian/customization.markdown]: Typo, `SHH`

File: source/_docs/installation/hassbian/upgrading.markdown [hunk 3]
```diff
@@ -20,6 +20,9 @@ $ sudo apt-get -y upgrade
 
 
 #### {% linkable_title Updating Home Assistant %}
+<p class='note'>
+You can aslo so the atomated way with "hassbian-config" by running "sudo hassbian-config upgrade home-assistant"
```
Comment (fabaff) [inline source/_docs/installation/hassbian/upgrading.markdown]: Fix the typos and please don't remove the formatting.

**Comment by ludeeus** [comment]
@fabaff Typo's corrected, not sure what you mean by "don't remove the formatting"?

**Comment by fabaff** [comment]
Commands should be marked as `code`.

**Comment by Landrash** [comment]
Fixed the commands that where supposed to be code @ludeeus . Thanks for fixing this!

**Comment by fabaff** [comment]
@ludeeus Thanks  for your contribution.

**Pull request merged**